lib/claim/token-holder-revenue-fund-contracts-ensemble.js

```diff
@@ -269,13 +286,16 @@ class TokenHolderRevenueFundContractsEnsemble {
         if (!isSpanned.call(this, currency))
             await span.call(this, currency);
 
-        const ensemble = _ensemble.get(this);
+        const configurations = _configurations.get(this);
         const currencyKey = currencyToKey(currency);
 
         try {
             const txs = [];
 
-            for (const configuration of ensemble) {
+            for (const configuration of configurations) {
+                if (!configuration.spansByCurrency || !configuration.spansByCurrency.has(currencyKey))
+                    continue;
```

Comment:
Is it not the the case that `await span.call(...)` above ensures that span exists?
If the span is ensured to exist, do we need this test here?
Same comment goes for the other two copies of the same check.

Reply:
True